doc/configuration.md

```diff
@@ -43,3 +43,28 @@ Can survive restart. Still needs some other method to act as cache.
 3scale hosts thousands of gateways for its customers and needs a reasonable way to share resources between them. Multi-tenant deployment of this proxy is the preferred way.
 
 TODO: figure out how to store/load the configuration in multi-tenant way
+
+
+## How config is loaded:
+
+On APICast there are three main components regarding configuration, the
+components are the following ones:
+
+- configuration_loader: this is a object that retrieve the configuration and the
+  OIDC config for all services, there are a few kinds of loader:
+    - file: for using local file and THREESCALE_CONFIG_FILE
+    - remote_v1: this is what we used in the past, to load all the config from
```

Comment:
Does remote_v1 really use `THREESCALE_CONFIG_FILE`? Seems a bit misleading worded like that because APIcast will not make that request to system if the env var `THREESCALE_CONFIG_FILE` is configured. I think you are confusing the "use case" for `THREESCALE_CONFIG_FILE` where a user can call the same API used by remote_v1 to retrieve a single config file to store on some local path. APIcast will only try remote_v1 if requests via remote_v2 fail to retrieve a config, it's a fallback. This only happens when a 3scale account still has proxy config configured the old way, I think we actually force migrated everyone last year or 2019 and so there will be no APIcast deployment using this endpoint anymore (we can check Kibana to be sure)

Reply:
yes, a typo!